codec/test_codec.go

```diff
@@ -86,30 +89,35 @@ type MyInnerStruct3 struct {
 	F   Foo           `serialize:"true"`
 }
 
+type MyStructWithPtr struct {
+	N1 *int32 `serialize:"true,omitempty"`
+	N2 *int64 `serialize:"true,omitempty"`
+}
+
 type myStruct struct {
-	InnerStruct  MyInnerStruct               `serialize:"true"`
+	InnerStruct  MyInnerStruct               `serialize:"true,omitempty"`
```

Comment:
I wonder if it would be better if we keep these without `omitempty` and add new types with `omitempty`.